jdk/src/share/back/export/sys.h

```diff
@@ -37,7 +37,7 @@
 
 /* Implemented in linker_md.c */
 
-void    dbgsysBuildLibName(char *, int, const char *, const char *);
+void    dbgsysBuildLibName(char *, size_t, const char *, const char *);
```

Comment:
This file needs the IBM copyright added.

Reply:
Done.